settings.gradle

```diff
@@ -23,11 +23,34 @@
  * questions.
  */
 
-include "base", "graphics", "controls", "swing", "swt", "fxml", "jsobject", "web", "media", "systemTests"
+include "base",
+        "graphics",
+        "controls",
+
+        // Add an entry for each incubator module here, leaving the incubator
+        // placeholder lines as an example.
+        // BEGIN: incubator placeholder
+        //"incubator.mymod",
+        // END: incubator placeholder
+
+        "swing",
+        "swt",
+        "fxml",
+        "jsobject",
+        "web",
+        "media",
+        "systemTests"
```

Comment:
thank you for placing each entry on a separate line!

I would prefer to remove the blank lines around the comment so as not to break the visual grouping (the comments are typically syntax colored anyway unless the user is on TRS80)

Reply:
OK